src/globals.ts

```diff
@@ -29,21 +29,36 @@ const warnSpansSizeOnce = runOneTimeWrapper((threshold: number, currentSize: num
     `Lumigo tracer is no longer collecting data on the invocation - maximum size of total spans collected (${threshold} bytes allowed, current size is ${currentSize} bytes)`
   );
 }, {});
+export enum droppedSpanReasons {
```

Comment:
Usually the JS convention is `DroppedSpanReasons`

Reply:
Fixed